scripts/add_qm_metrics.py

```diff
@@ -25,9 +25,7 @@
 )
 args = parser.parse_args()
 db = mc.connect_to_mc_db(args)
-session = db.sessionmaker()
 
-for f in args.files:
-    session.ingest_metrics_file(f, args.type)
-
-session.commit()
+with db.sessionmaker() as session:
+    for f in args.files:
+        session.ingest_metrics_file(f, args.type)
```

Comment:
I think you still need the commit, it doesn't look like that method has a commit at the end.

Reply:
The context manager has a commit on __exit__